CMakeLists.txt

```diff
@@ -66,15 +84,21 @@ target_link_libraries(prime_workerd prime_server ${CMAKE_THREAD_LIBS_INIT})
 set_target_properties(prime_server PROPERTIES PUBLIC_HEADER "${PRIME_LIBRARY_HEADERS}")
 
 install(TARGETS prime_server
-	LIBRARY DESTINATION /usr/local/lib
-	PUBLIC_HEADER DESTINATION /usr/local/include)
+	LIBRARY DESTINATION "lib"
+	PUBLIC_HEADER DESTINATION "include/prime_server")
 
 install(TARGETS prime_echod
 	prime_filed
 	prime_httpd
 	prime_serverd
 	prime_proxyd
-	prime_workerd DESTINATION /usr/local/bin)
+	prime_workerd DESTINATION "bin")
+
+CONFIGURE_FILE(
```

Comment:
can we generate `libprime_server.pc` here instead of using `libprime_server.pc.in` that way we dont break autotools?